src/Context/DrupalFormContext.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @file
+ * The DrupalFormContext behat context.
+ */
+
+namespace drunomics\BehatDrupalUtils\Context;
+
+/**
+ * Defines application features from the specific context.
+ */
+class DrupalFormContext extends DrupalFormContextBase {
+
+  /**
+   * @When I click :tab in user menu
+   */
+  public function iClickInUserMenu($tab) {
```

Comment:
Is this necessary? I did the same in contentpool-filter or basic tests with "I click Edit in element ".tab"". Anyway, I'm fine with keeping it, but then: This is not the user menu. As code shows, it are the local tasks. Thus it should be renamed accordingly.